backend/tests/test_project.py

```diff
@@ -19,7 +19,7 @@ def tearDown(self):
         db.drop_all()
         self.app_context.pop()
 
-    def test_project_routes(self):
+    def test_update_project(self):
```

Comment:
Tbh next time you could just make a separate PR for this change, in practice, each PR should be scoped to it's own problem so adding random changes here and there makes it harder for reviewers to follow

Reply:
Will do